go/types/encode_human_readable_test.go

```diff
@@ -406,9 +406,9 @@ func TestEmptyCollections(t *testing.T) {
 	c := MakeMapType(BlobType, NumberType)
 	assertWriteTaggedHRSEqual(t, "Type(Map<Blob, Number>)", c)
 	d := NewMap()
-	assertWriteTaggedHRSEqual(t, "Map<, >({})", d)
+	assertWriteTaggedHRSEqual(t, "Map<, >", d)
```

Comment:
This was the output I didn't like. I would prefer it to print `Map<>({})` or `Map<,>({})`.

I think we should always print the value.